### PR TITLE
Adding Design

### PR DESCRIPTION
**Pull request by SnakyMz**
Things I have done:

- [ ] Added layout for calculator
- [ ] Added Child component for calculator
- [ ] Added CSS

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      </div>
+    </section>
+  );
+}
+
+function Display() {
+  return (
+    <span className="display">0</span>
+  );
+}
```
Comment (fobadara) [inline src/components/Calculator.js:34]: [OPTIONAL] I think you should move this child component to an external file. This helps with the separation of concerns.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        <span className="grey">5</span>
+        <span className="grey">6</span>
+        <span className="orange">-</span>
+        <span className="grey">1</span>
+        <span className="grey">2</span>
+        <span className="grey">3</span>
+        <span className="orange">+</span>
+        <span className="grey zero">0</span>
+        <span className="grey">.</span>
+        <span className="orange">=</span>
```
Comment (fobadara) [inline src/components/Calculator.js:24]: [OPTIONAL] I think you should move this to an external component. This helps with the separation of concerns.

**Review by fobadara — APPROVED** [review]
Hi @SnakyMz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ clean code
✅ Good file structure

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (SnakyMz) [reply src/components/Calculator.js:34]: Thank you, I will make changes as suggested.

File: src/components/Calculator.js (hunk 2 above)
Comment (SnakyMz) [reply src/components/Calculator.js:24]: Thank you, will make changes as suggested.

**Pull request merged**